### DISCUSSION TITLE
Sort on General Search

### DISCUSSION THREAD

**Discussion by ghaddon** [dailydotdev/daily#658] [Feature Requests]
I would like to request a sort capibility on general search. This would provide for found content to be presented in date sequence from newest to oldest and oldest to newest.

An example would be looking to sort content return for key word Flutter.  Right now all content is ordered from oldest to newest.

Thanks, Gary

**Comment by idoshamun** [comment]
Hi @ghaddon,

Thanks for the suggestion. This is a dup of https://github.com/dailydotdev/daily/discussions/294. Let's keep the discussion there :)
